### ISSUE TITLE
Record Pictures and Videos with ScreenReader VoiceOver not accessible

### ISSUE THREAD

**Issue by sebastian2012** [signalapp/Signal-iOS#5380]
Hi,
the Picture Record Window is not Accessible with VoiceOver. The buttons havn't any Labels and are not Pressable with ScreenReaders.
